autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -252,7 +259,7 @@ def fit(train_data, label, tuning_data=None, output_directory=None, problem_type
         scheduler_options = {
             'resource': {'num_cpus': nthreads_per_trial, 'num_gpus': ngpus_per_trial},
             'num_trials': num_trials,
-            'time_out': time_limits,
+            'time_out': time_limits_hpo,
```

Comment:
Current implementation of the scheduler doesn't support real time_out. It does not stop `scheduled trials`.

Reply:
Yes, hopefully it does not go too far beyond the time limit though. In future we can investigate forcing model early stopping to ensure runtime stays below time_limits.